fe/fe-core/src/main/java/com/starrocks/catalog/DictionaryMgr.java

```diff
@@ -652,32 +636,12 @@ private void commit() {
             }
             dictionary.setCommitting();
 
-            PRefreshDictionaryCacheCommitRequest request = new PRefreshDictionaryCacheCommitRequest();
+            PProcessDictionaryCacheRequest request = new PProcessDictionaryCacheRequest();
             request.dictId = dictionary.getDictionaryId();
             request.txnId = txnId;
+            request.type = PProcessDictionaryCacheRequestType.COMMIT;
 
-            for (TNetworkAddress address : beNodes) {
-                PRefreshDictionaryCacheCommitResult result = null;
-                try {
-                    Future<PRefreshDictionaryCacheCommitResult> future =
-                            BackendServiceClient.getInstance().refreshDictionaryCacheCommit(address, request);
-                    result = future.get();
-                } catch (Exception e) {
-                    errMsg = e.getMessage();
-                    error = true;
-                    return;
-                }
-
-                TStatusCode code = TStatusCode.findByValue(result.status.statusCode);
-                if (code != TStatusCode.OK) {
-                    LOG.warn(" RefreshDictionaryCacheCommit failed in: " + address + " err " + result.status.errorMsgs);
-                    errMsg = result.status.errorMsgs.size() == 0 ? "" : result.status.errorMsgs.get(0);
-                    error = true;
-                    return;
-                }
-            }
-
-            LOG.info("finish RefreshDictionaryCacheCommit dictionary id: {}, txn id: {}", request.dictId, request.txnId);
+            error = DictionaryMgr.processDictionaryCacheInteranl(request, errMsg, beNodes, null);
```

Comment:
error is not handled?

Reply:
It will handle in another place, not here